config/routes.rb

```diff
@@ -48,76 +47,85 @@
     get '/interstitial', to: 'after_sign_in#interstitial', as: :interstitial
 
     scope '/application' do
-      get '/review/submitted/:id' => 'application_form#review_previous_application', as: :review_previous_application
-
-      get '/' => 'unsubmitted_application_form#show', as: :application_form
       get '/prefill', to: 'prefill_application_form#new'
       post '/prefill', to: 'prefill_application_form#create'
-      get '/review' => 'unsubmitted_application_form#review', as: :application_review
+
       get '/before-you-start', to: 'unsubmitted_application_form#before_you_start'
+      get '/' => 'unsubmitted_application_form#show', as: :application_form
+      get '/review' => 'unsubmitted_application_form#review', as: :application_review
       get '/submit' => 'unsubmitted_application_form#submit_show', as: :application_submit_show
       post '/submit' => 'unsubmitted_application_form#submit', as: :application_submit
 
+      get '/submit-success' => 'submitted_application_form#submit_success', as: :application_submit_success
       get '/complete' => 'submitted_application_form#complete', as: :application_complete
       get '/review/submitted' => 'submitted_application_form#review_submitted', as: :application_review_submitted
-      get '/submit-success' => 'submitted_application_form#submit_success', as: :application_submit_success
-      get '/edit' => 'submitted_application_form#edit', as: :application_edit
+      get '/review/submitted/:id' => 'application_form#review_previous_application', as: :review_previous_application
+
       get '/start-apply-again' => 'submitted_application_form#start_apply_again', as: :start_apply_again
       post '/apply-again' => 'submitted_application_form#apply_again', as: :apply_again
+
       get '/start-carry-over' => 'carry_over#start', as: :start_carry_over
       post '/carry-over' => 'carry_over#create', as: :carry_over
 
-      scope '/personal-details' do
+      scope '/personal-information' do
         get '/' => 'personal_details/base#new', as: :personal_details
         patch '/' => 'personal_details/base#create'
         get '/edit' => 'personal_details/base#edit', as: :edit_personal_details
         patch '/edit' => 'personal_details/base#update'
-        get '/nationalities' => 'personal_details/nationalities#new', as: :nationalities
-        patch '/nationalities' => 'personal_details/nationalities#create'
-        get '/nationalities/edit' => 'personal_details/nationalities#edit', as: :edit_nationalities
-        patch '/nationalities/edit' => 'personal_details/nationalities#update'
+
+        get '/nationality' => 'personal_details/nationalities#new', as: :nationalities
+        patch '/nationality' => 'personal_details/nationalities#create'
+        get '/nationality/edit' => 'personal_details/nationalities#edit', as: :edit_nationalities
+        patch '/nationality/edit' => 'personal_details/nationalities#update'
+
```

Comment:
The page title is ‘What is your nationality?’ and the review label is ‘nationality’.